src/components/select/editForm/Select.edit.data.js

```diff
@@ -648,4 +648,19 @@ export default [
     tooltip: 'A raw JSON object to use as options for the Select component (Choices JS).',
     defaultValue: {},
   },
+  {
+    type: 'checkbox',
+    input: true,
+    weight: 29,
+    key: 'allowCaching',
```

Comment:
This is not reverse compatible. We need to make this falsey by default. I propose to call it ```ignoreCache``` and then make the setting called the same. This way you could just set the following and it will be reverse compatible.

```
options.ignoreCache = this.component.ignoreCache;
```

Reply:
I set it to true because by default we are always using cache for Select components and if  set it to false, all the Selects will 'disallowCaching'.
But your suggestion is much clear

Reply:
The problem arises with forms that were created before this change and do not have the ```allowCaching``` property set. Those forms will now start ignoring cache after the update which will break reverse compatibility.